android/app/src/main/java/org/fossasia/openevent/adapters/DayScheduleAdapter.java

```diff
@@ -99,6 +105,45 @@ public void onBindViewHolder(DayScheduleViewHolder holder, int position) {
             holder.slotTrack.getBackground().setColorFilter(storedColor, PorterDuff.Mode.SRC_ATOP);
             holder.slotTrack.setText(sessionTrack.getName());
 
+            if(currentSession.isBookmarked()) {
```

Comment:
I'm talking about this code 

https://github.com/shreyasnbhat/open-event-android/blob/f1e4b5856cc3402879eb6c1dce85dabe9f3a4e46/android/app/src/main/java/org/fossasia/openevent/adapters/DayScheduleAdapter.java#L108-L145

It repeats a lot